### ISSUE TITLE
Branch in git for changes

### ISSUE THREAD

**Issue by nickwood2009** [coleam00/bolt.new-any-llm#113]
I think this would be more related to the agents system feature request. when you spawn an agentic process, the agents should work on a branch, the end result being a branched feature to test, after fixes etc, Create a PR to merge into main. Can keep many features in separate "Threads" running in parallel then.

**Comment by chrismahoney** [comment]
👍
